packages/react-app/src/components/giveaways/cards/Card.tsx

```diff
@@ -53,7 +53,13 @@ const GiveawayCard = ({
     if (winners.length === 0) return;
 
     const winnerId = winners[0].id;
-    const winnerParticipant = participants?.find((p) => p.id === winnerId);
+    let winnerParticipant;
+
+    if (userInfo.role === UserRole.ADMIN) {
+      winnerParticipant = participants.find((p) => p.id === winnerId);
+    } else {
+      winnerParticipant = participants.pop();
+    }
```

Comment:
We can remove this part, we don't need the participants anymore to get the winners name

Reply:
We can also remove:
`const { data: participants } = useParticipants(giveaway._id);`
Participants are no longer used on this component